operators/deploy/custom-metrics-server/values.yaml

```diff
@@ -0,0 +1,58 @@
+# Default values for custom-metrics-server.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: crownlabs/custom-metrics-server
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart version.
+  tag: ""
+
+configurations:
+  runtimeEndpoint: unix:///run/containerd/containerd.sock
+  connectionTimeout:  10
+  updatePeriod: 1
+  grpcPort: 9099
+
+nameOverride: ""
+fullnameOverride: ""
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  fsGroup: 0  # Cluster root group-id
+  runAsGroup: 0
+  runAsNonRoot: false
+  runAsUser: 0
```

Comment:
The cms needs read-access to /run/containerd/containerd.sock. 
`containerd.sock` is usually owned by root:root and its permissions mode is 660. 

Maybe it could be enough to add cms user to **root** fsGroup, since root group has read rights. I will check it.

Reply:
I confirm that if the cms user is added to group 0(root), the security context can be set to run as non-root.